v3-nginx/Dockerfile

```diff
@@ -84,34 +89,33 @@ COPY --from=build /usr/share/TLS/server.key /etc/nginx/conf/server.key
 COPY --from=build /usr/share/TLS/server.crt /etc/nginx/conf/server.crt
 COPY ./tls.conf /etc/nginx/conf.d/tls.conf.disabled
 
-RUN export version=$(echo `/usr/sbin/nginx -v 2>&1` | cut -d '/' -f 2) && \
-    wget --quiet http://nginx.org/download/nginx-$version.tar.gz && \
-    tar -xvzf nginx-$version.tar.gz && \
-    cd /nginx-$version && \
+RUN version="$(/usr/sbin/nginx -v 2>&1 | cut -d '/' -f 2)" && \
+    wget --quiet http://nginx.org/download/nginx-"$version".tar.gz && \
+    tar -xvzf nginx-"$version".tar.gz && \
+    cd /nginx-"$version" && \
     ./configure --with-compat --add-dynamic-module=../ModSecurity-nginx && \
     make modules && \
     cp objs/ngx_http_modsecurity_module.so /etc/nginx/modules
 
 RUN mkdir /etc/modsecurity.d && \
-    cd /etc/modsecurity.d && \
-    echo "include \"/etc/modsecurity.d/modsecurity.conf\"" > setup.conf && \
-    wget --quiet https://raw.githubusercontent.com/SpiderLabs/ModSecurity/v3/master/modsecurity.conf-recommended && \
-    mv modsecurity.conf-recommended modsecurity.conf && \
-    wget --quiet https://raw.githubusercontent.com/SpiderLabs/ModSecurity/v3/master/unicode.mapping
+    echo "include \"/etc/modsecurity.d/modsecurity.conf\"" > /etc/modsecurity.d/setup.conf && \
+    wget --quiet https://raw.githubusercontent.com/SpiderLabs/ModSecurity/v3/master/modsecurity.conf-recommended \
+      -o /etc/modsecurity.d/modsecurity.conf && \
+    wget --quiet https://raw.githubusercontent.com/SpiderLabs/ModSecurity/v3/master/unicode.mapping \
+      -o /etc/modsecurity.d/unicode.mapping
 
 RUN sed -i '1iload_module modules/ngx_http_modsecurity_module.so;' /etc/nginx/nginx.conf && \
     sed -i -e 's/http {/http {\n    modsecurity on;\n    modsecurity_rules_file \/etc\/modsecurity.d\/setup.conf;\n/g' /etc/nginx/nginx.conf
 
 ENV LD_LIBRARY_PATH /lib:/usr/lib:/usr/local/lib
 
-RUN apt-get install -y vim
-RUN echo $SERVERNAME
+RUN echo "$SERVERNAME"
```

Comment:
I'm not sure I understand this line. That would add an image layer just for printing out the `SERVERNAME` env variable? Does that make sense?

Reply:
No, but it was there :) Removing.